datasets/new_york/pipelines/new_york/pipeline.yaml

```diff
@@ -0,0 +1,606 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+resources:
+
+  - type: bigquery_table
+    table_id: new_york
+    description: "New York Pipelines"
+
+dag:
+  airflow_version: 2
+  initialize:
+    dag_id: new_york
+    default_args:
+      owner: "Google"
+      depends_on_past: False
+      start_date: '2021-03-01'
+    max_active_runs: 1
+    schedule_interval: "@daily"
+    catchup: False
+    default_view: graph
+
+  tasks:
+    - operator: "GKECreateClusterOperator"
+      args:
+        task_id: "create_cluster"
+        project_id: "{{ var.value.gcp_project }}"
+        location: "us-central1-c"
+        body:
+          name: new-york
+          initial_node_count: 4
+          network: "{{ var.value.vpc_network }}"
+          node_config:
+            machine_type: e2-standard-16
+            oauth_scopes:
+              - https://www.googleapis.com/auth/devstorage.read_write
+              - https://www.googleapis.com/auth/cloud-platform
+
+    - operator: "GKEStartPodOperator"
+      description: "Run New York 311 Service Requests Pipeline"
+      args:
+        task_id: "transform_csv_ny_311_service_requests"
+        name: "311_service_requests"
+        project_id: "{{ var.value.gcp_project }}"
+        location: "us-central1-c"
+        cluster_name: new-york
+        namespace: "default"
+        image_pull_policy: "Always"
+        image: "gcr.io/bigquery-public-data-dev/new_york__run_csv_transform_kub_311_service_requests:latest"
```

Comment:
The container image URL should be parameterized (and not exposed). Our prod environment has a different URL, you're using the dev one here.

Reply:
Done